Makefile

```diff
@@ -59,9 +62,15 @@ check: bin/golangci-lint ## Check code quality (lint)
 	./bin/golangci-lint run --timeout 300s
 	cd test/_e2e && ../../bin/golangci-lint run --timeout 300s
 
+checkstyle: tools/goimports ## Check code style (imports and formatting)
```

Comment:
There is already a directory for binaries necessary for testing: `./bin`.  Perhaps use that directory rather than a new `tools` directory?